embc-app/DataInterfaces/CommunityExtensions.cs

```diff
@@ -19,6 +19,7 @@ public static void UpdateCommunity(this EmbcDbContext context, Community Communi
         {
             Community _Community = context.Communities.FirstOrDefault<Community>(x => x.Id == Community.Id);
```

Comment:
bad naming convention?

Reply:
Shoot, I missed the upper case, so yes bad naming convention.  Thx.  Shall be changed